### DISCUSSION TITLE
What are the files and folders in storage/statamic/ used for?

### DISCUSSION THREAD

**Discussion by kolaente** [statamic/cms#5874] [Q&A] [answered]
Hi!

I've recently come across quite a few files in the `storage/statamic/` directory. I'm not quite sure what to make of these. Should they be checked into the git repo? Can I delete them? Should I add them to the gitignore?

I figured `storage/statamic/glide` contains only cached images but what about the rest?

My main problem with these is they seem to prevent pulling new changes on production:

```
$ git pull
error: The following untracked working tree files would be overwritten by merge:
	storage/statamic/password_resets/password_activations.yaml
	storage/statamic/password_resets/password_resets.yaml
	storage/statamic/search/default.json
	storage/statamic/stache-locks/sf.stache-warming.5R9ZBci.lock
	storage/statamic/tokens/84jSLRTiI3aBMMsRi3CF1RjaQdFjq7z73qewkO19.yaml
	storage/statamic/tokens/FWVntsEv4k3d191oknhA3lmEBpFWmLB0VK2TRvtR.yaml
	storage/statamic/tokens/YmryoIVe6PiZtIyy12x5DbI40zbAjaAu6Z6dBnXY.yaml
	storage/statamic/tokens/Z7zvb7Ji4pd3t1yq0h2TQEbUd4AWAjdCCdcO6Fm2.yaml
	storage/statamic/users/22f99b45-dbf1-4da5-a0b8-32bfabf230ab.yaml
	storage/statamic/users/28e1b6d1-4955-4f52-8791-70cac8ecb301.yaml
	storage/statamic/users/86ed58e8-70fe-4db7-a462-1d1589f63622.yaml
	storage/statamic/users/baa21642-b05b-401e-b053-a644f7594c01.yaml
	storage/statamic/users/eadff8bd-4094-4611-a2c5-a1f500c712c5.yaml
Please move or remove them before you merge.
Aborting
```

**Comment by afonic** [comment, marked as answer]
They are temp files that should not be in version control.

If you see the `statamic/statamic` repo, this whole folder can safely be ignored: https://github.com/statamic/statamic/tree/master/storage/statamic

You probably deleted or misplaced the `.gitignore` file (or used another install method).

**Reply by kolaente** [reply]
Awesome, thanks for the clarification!
